### PR TITLE
Add screen recording

### PR DESCRIPTION
**Pull request by schorliee**
Add screen recording + save file in download directory

### CODE REVIEW

**Comment by alejo096** [comment]
@schorlie @abdulmth I think it could be a good idea to delete the IDE files from the pull request. I think we don't really need them.

**Review by alejo096 — APPROVED** [review]
Looks good to me! Thank you

**Comment by lbb** [comment]
@abdulmth @alejo096 @schorlie, Please don't delete things unrelated to code review. Create a ticket, and we can discuss deleting the config folder. I'll add back the `.idea` folder and we can discuss this later.

**Comment by lbb** [comment]
Please delete the current working branch after a merge.

File: amos-tub-ws18-proj1.iml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lbb) [inline amos-tub-ws18-proj1.iml:1]: How can this file be in your PR? It is stated in the `.gitignore` file of our project, please run `git rm -r --cached .`

**Pull request merged**